src/tools/miri/tests/panic/mir-validation.rs

```diff
@@ -8,6 +8,12 @@
 // Somehow on rustc Windows CI, the "Miri caused an ICE" message is not shown
 // and we don't even get a regular panic; rustc aborts with a different exit code instead.
 //@ignore-host: windows
+
+// FIXME: output is different between stage1 and stage2. This should be possible to remove
+// when stages get bumped.
+// For the grep: cfg(bootstrap)
+//@normalize-stderr-test: "thread 'rustc' panicked" -> "thread 'rustc' ($$TID) panicked"
```

Comment:
Is this still true after the stage0 redesign? Which library is getting used here?

Reply:
Just to confirm from testing locally; if I remove the normalize in `track.rs`, `./x test tests/ui/track-diagnostics/track.rs` fails but `./x test --stage 2 tests/ui/track-diagnostics/track.rs` succeeds. I _think_ this one failed in CI as well but I don't recall for sure.

The Miri test I know for sure failed both locally and in CI without the workaround.

Reply:
Oh, wait, this makes sense doesn't it; everything that is a panic during runtime is using the in-tree compiler, so the output change will show up in the tests immediately. But the two tests here are panics in rustc at comptime, and that rustc was built using downloaded std so it won't have the update - at least until stage 2.

So having the normalize step here is expected to be needed (or somehow skipping the test for stage1), I was just getting myself confused around the redesign and thinking it shouldn't be necessary.

Reply:
Updated the comment to make this more clear